### PR TITLE
Assassin’s Creed II

### PR DESCRIPTION
**Pull request by plata**

### CODE REVIEW

**Comment by plata** [comment]
@qparis how did you get Uplay working for AC: Brotherhood?

File: Games/Assassin’s Creed II/Steam/script.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"scriptName": "Steam",
+	"compatibleOperatingSystems": ["LINUX"],
+	"testingOperatingSystems": ["MACOSX"],
```
Comment (qparis) [inline Games/Assassin’s Creed II/Steam/script.json]: Cannot be testing if it is not compatible :)

**Comment by qparis** [comment]
Should work as it should'nt it?

**Comment by plata** [comment]
I thought so as well. But Uplay keeps crashing at start. So I thought you should have the same problem with AC: Brotherhood.

**Comment by Brainzyy** [comment]
@plata @qparis Just a heads up, I heard that a recent UPlay update broke compatibility with wine.

**Comment by plata** [comment]
@Brainzyy thanks for the remark. I'm trying now to write a Uplay script nevertheless.

**Comment by Brainzyy** [comment]
@plata You're welcome, you can still login to UPlay with a prefix set to Windows Vista and latest Wine Staging, but according to that bug, games don't work anymore.

**Comment by qparis** [comment]
Agreed, we should remove AC:B for the moment

**Comment by plata** [comment]
@qparis just set it to testing. Would be a shame if we would lose the script completely.

**Comment by qparis** [comment]
Maybe we should also create a UPlaySteamScript template

**Comment by plata** [comment]
Yes. Also have been thinking about this. Just wait a sec I will push my UPlay script.

**Comment by Brainzyy** [comment]
@plata @qparis A seperate UplayScript would be useful as well. Since some games are not on steam, but only on Uplay.

**Comment by plata** [comment]
@Brainzyy I think it is not possible to start a Uplay installation from a script.

**Comment by plata** [comment]
Uplay does not crash anymore. However, it's still not possible to play the game. Uplay shows "There's a problem connecting to the internet..." and "Unable to download Uplay game files". 
Maybe this is related to
```
fixme:file:MoveFileWithProgressW MOVEFILE_WRITE_THROUGH unimplemented
```
or
```
fixme:crypt:CRYPT_VerifyChainRevocation Unable to find CRL for CA certificate
```

**Comment by plata** [comment]
The script itself is fine and can be merged. The problem is #221.

**Pull request merged**